kebechet/managers/update/update.py

```diff
@@ -235,62 +235,6 @@ def _get_direct_dependencies_requirements(req_dev: bool) -> set:
 
         return direct_dependencies
 
-    def _get_all_packages_versions(self) -> dict:
-        """Parse Pipfile.lock file and retrieve all packages in the corresponding locked versions."""
-        try:
-            with open("Pipfile.lock") as pipfile_lock:
-                pipfile_lock_content = json.load(pipfile_lock)
-        except Exception as exc:
-            # TODO: open a PR to fix this
-            raise DependencyManagementError(
-                f"Failed to load Pipfile.lock file: {str(exc)}"
-            ) from exc
-
-        result = {}
-
-        for package_name, package_info in pipfile_lock_content["default"].items():
-            if "git" in package_info:
-                self._create_unsupported_package_issue(package_name, "git")
-                raise DependencyManagementError(
-                    "Failed to find version in package that uses git source."
-                )
-            elif "path" in package_info:
-                self._create_unsupported_package_issue(package_name, "local")
-                raise DependencyManagementError(
-                    "Failed to find version in package that uses local source."
-                )
-            result[package_name.lower()] = {
-                "dev": False,
-                "version": package_info["version"][len("==") :],
-            }
-
-        for package_name, package_info in pipfile_lock_content["develop"].items():
-            if "git" in package_info:
-                self._create_unsupported_package_issue(package_name, "git")
-                raise DependencyManagementError(
-                    "Failed to find version in package that uses git source."
-                )
-            elif "path" in package_info:
-                self._create_unsupported_package_issue(package_name, "local")
-                raise DependencyManagementError(
-                    "Failed to find version for package that uses local installation."
-                )
-            result[package_name.lower()] = {
-                "dev": False,
-                "version": package_info["version"][len("==") :],
-            }
-        # Close git as a source issues.
-
-        issue = self.get_issue_by_title(
-            _ISSUE_UNSUPPORTED_PACKAGE.format(env_name=self.runtime_environment)
```

Comment:
The definition of that message should also be removed if it is not used anymore

https://github.com/thoth-station/kebechet/blob/a7c2f9d6c207cb3740b268fd2502c0bdb6c15bdf/kebechet/managers/update/update.py#L77-L80

Reply:
ack, thanks